frameworks/ember/README.md

```diff
@@ -3,14 +3,28 @@
 Below are links to tools that will help you use the [ArcGIS API for JavaScript] in applications built with [Ember], as well as examples of how to use those tools.
 
 ## Reusable Libraries
+Esri uses the tools below to to integrate the [ArcGIS API for JavaScript] into [Ember] applications like [ArcGIS Hub](https://hub.arcgis.com/) and [Operations Dashboard for ArcGIS](https://doc.arcgis.com/en/operations-dashboard/).
+
 ### [ember-cli-amd](https://github.com/Esri/ember-cli-amd)
 An [EmberCLI](https://ember-cli.com/) addon for using AMD libraries that helps you load modules from the ArcGIS API for JavaScript (either 3.x or 4.x) in an Ember application.
 <br />[Blog post](http://odoe.net/blog/ember-with-arcgis-api-for-javascript/) | [Video](https://www.youtube.com/watch?v=9wbGhr6wdwE)
 
+### [ember-esri-loader](https://github.com/Esri/ember-esri-loader/)
+An [Ember](https://ember-cli.com/) addon to allow lazy loading the ArcGIS API for JavaScript in Ember applications.
+<br />[View it live](http://ember-esri-loader.surge.sh/)
+
+### [ember-arcgis-server-services](https://github.com/Esri/ember-arcgis-server-services)
+A set of Promise based Ember services for working with ArcGIS Server services
+<br />[View it live](http://ember-arcgis-server-services.surge.sh/)
+
+### [ember-arcgis-portal-services](https://github.com/Esri/ember-arcgis-portal-services)
+Set of promise-based, Ember Services for working with ArcGIS Enterprise API
+<br />[View it live](https://eapsprod.surge.sh)
+
 ## Example Applications
 
 ### [esrijs4-vm-ember](https://github.com/odoe/esrijs4-vm-ember)
-Using ErsiJS 4.0 View Models with Ember
+Using ErsiJS 4.0 View Models with [Ember]
```

Comment:
@odoe - should we still be linking to this?

Reply:
We can remove those links now